pom.xml

```diff
@@ -338,6 +345,24 @@
     </distributionManagement>
 
     <profiles>
+        <profile>
+            <id>java11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Dumb maven question: why do we need a java11 profile?  to specify a different compiler plugin?

Reply:
It's to set the maven-compiler-plugin's `release` configuration property. Annoyingly this is not recognized by the Java 8 compiler, so it can only be set in the Java 11 configuration. Using a profile was the only way I could see to do this.